doc-content/enterprise-only/admin-and-config/managing-business-central-user-management-con.adoc

```diff
@@ -0,0 +1,4 @@
+[id='managing-business-central-user-management-con']
+= User management
+
+You can use {CENTRAL} to create and manage as many users and groups as you require. A user must be assigned at least one role to login to {CENTRAL}. User privileges depend on the groups the user is a member of and the roles assigned to the groups.
```

Comment:
_A user must be assigned at least one **user-specific** role to login to {CENTRAL}._
I would emphasize this as there are roles (such as _rest-all_) which can be assigned to a user but does not grant the login rights as well.

Reply:
changed.